contracts/instruments/RibbonThetaVault.sol

```diff
@@ -278,6 +282,51 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
         return amountAfterFee;
     }
 
+    /**
+     * @notice Lock's users shares for future withdraw and ensures that the new short excludes the scheduled amount.
+     * @param shares is the number of shares to be withdrawn in the future.
+     */
```

Comment:
we don't want to allow user to schedule withdraw for x and then decide he wants to make it 2x before the friday?

Reply:
currently no. User's schedule their withdraw amount, and that is fixed until they complete the withdraw